### ISSUE TITLE
Offer search options (not results) when a user starts to type in the search box

### ISSUE THREAD

**Issue by davidbowdley** [runbox/runbox7#788]
With search it isn't always easy to find what you are looking for and a refined search is more difficult to do than it needs to be. 

For example, if you start typing a search term in some interfaces it begins to offer options on where it finds that term. A drop down option could appear so you can choose where you are looking for that search term. Options might be To, From, Subject or Body depending on what the system found in the index. If you choose one of those you can then continue to type so you can look for a second term in the same search. Effectively becoming an AND or OR search as needed.

**Comment by gtandersen** [comment]
Good idea. There could also be an option to select the current or all folders, as that's often used with searches as well.

**Comment by davidbowdley** [comment]
@gtandersen Yes, forgot to mention that additional option! :)
